### ISSUE TITLE
Elasticsearch data source - template variables seem not to be updated properly when entering dashboard

### ISSUE THREAD

**Issue by machytkafitanalytics** [grafana/grafana#14000]
### What Grafana version are you using?
5.3.2 - but problem is also in 5.2.4 and versions between them

### What datasource are you using?
Elasticsearch 2.4.5

### What OS are you running grafana on?
Ubuntu 16.04

### What did you do?
Open dashboard which uses exclusively Elasticsearch data source

### What was the expected result?
Template variables should be properly updated based on Lucene queries

### What happened instead?
Only first template variable is updated. Second one has very strange values in its list and therefore dashboard shows only some data after entering it.
People have to manually select different value in the first template variable "shopname" and only after it second variable is properly updated and dashboard shows expected data.

Here is cut from screenshot to show strange content of the second template variables after entering dashboard:
![screenshot from 2018-11-08 12-20-53-x1](https://user-images.githubusercontent.com/17564930/48197297-a3861980-e355-11e8-804c-e955ff31d4d7.jpg)

Here is proper content after manual change in the first variable:
![screenshot from 2018-11-08 12-21-17-x1](https://user-images.githubusercontent.com/17564930/48197512-31fa9b00-e356-11e8-9df5-4cb1b2d99fd2.jpg)

Could you please look at it?
Thank you very much.

**Comment by marefr** [comment]
So you did not have these problem before 5.2.4, what version was that? I'm afraid you have to include your dashboard json model here for us to be able to test/verify. That is the json model as it is after you've loaded it in browser and haven't changed the first template variable. You can get the json model in dashboard settings and json model button. 

Other things that you can include, if possible:
* use chrome dev tools or similar and copy request payload of the template variable query when the dashboard loads. 

Please remove any sensitive information. Thank you
